beaglebone/servo/README.md

```diff
@@ -0,0 +1,47 @@
+﻿Code for Servos
+ ---
+### About
+
+This code is intended to run the servos in order to lower the beaker for science testing. The beaker should stay in place until an empty LCM struct is sent where it will lower the beaker.
+
+### LCM Channels Publishing/Subscribed To
+**ServoCMD**[subscriber] \
+Messages:  [ServoCMD.lcm](https://github.com/jjtom34/mrover-workspace/blob/AmmoniaServo/rover_msgs/ServoCmd.lcm) "/servocmd" \
+Publishers: tbd
+Subscribers: tbd
+
+
+### Usage
+#### Require Electronic Components:
+- 1 Servo
+- 1 Beaglebone Black board or similar board
+
+
+#### Setup
+When the servo is set up with the beaker, the program should be run as follows
+`$ ~/cd mrover-workspace`
+`$ /jarvis build beaglebone/ammoniaservo` \
```

Comment:
why wasn't this named "servo" or "science_servo"?

Reply:
it's servo in the code i hadn't updated the readme